op-chain-ops/deployer/pipeline/opchain.go

```diff
@@ -13,10 +13,10 @@ import (
 	"github.com/ethereum/go-ethereum/common"
 )
 
-// PermissionedGameStartingAnchorRoots is a root of `0xdead` for both game types at block 0, generated
-// with a console.log in DeployOpChain.s.sol
+// PermissionedGameStartingAnchorRoots is a root of `0xdead` for the permissioned game at block 0,
+// and no root for the permissionless game. It was generated with a console.log in DeployOpChain.s.sol
 var PermissionedGameStartingAnchorRoots = []byte{
-	0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x20, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x02, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x56, 0xd3, 0x26, 0xa8, 0xef, 0x15, 0x96, 0xf3, 0xd8, 0x73, 0x97, 0xc6, 0xc7, 0xb7, 0xb9, 0xda, 0xca, 0xf8, 0x68, 0x5e, 0xd9, 0x7d, 0xdd, 0x31, 0xf0, 0xaa, 0x04, 0x29, 0x4e, 0xaf, 0x9f, 0xca, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x01, 0x56, 0xd3, 0x26, 0xa8, 0xef, 0x15, 0x96, 0xf3, 0xd8, 0x73, 0x97, 0xc6, 0xc7, 0xb7, 0xb9, 0xda, 0xca, 0xf8, 0x68, 0x5e, 0xd9, 0x7d, 0xdd, 0x31, 0xf0, 0xaa, 0x04, 0x29, 0x4e, 0xaf, 0x9f, 0xca, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+	0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x20, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x01, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x01, 0x56, 0xd3, 0x26, 0xa8, 0xef, 0x15, 0x96, 0xf3, 0xd8, 0x73, 0x97, 0xc6, 0xc7, 0xb7, 0xb9, 0xda, 0xca, 0xf8, 0x68, 0x5e, 0xd9, 0x7d, 0xdd, 0x31, 0xf0, 0xaa, 0x04, 0x29, 0x4e, 0xaf, 0x9f, 0xca, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
```

Comment:
@mds1 Is this the abi encoded anchor roots array? It doesn't match what I expect for a 1-sized 0xdead root?

Reply:
Fixed in ca8be30